charts/minecraft/templates/deployment.yaml

```diff
@@ -239,6 +239,9 @@ spec:
         volumeMounts:
         - name: datadir
           mountPath: /data
+          {{- if (and .Values.persistence.dataDir.enabled .Values.persistence.dataDir.subPath) }}
+          subPath: {{ .Values.persistence.dataDir.subPath }}
```

Comment:
Should we also add a placeholder in the `values.yaml` file with an example (commented) of what this setting would be.

Generally speaking, if we reference a value, it's a good practice to have an example in the values.yaml file.

Reply:
Good point. Can you add that @gandazgul ?

Reply:
This is done